lmdeploy/serve/async_engine.py

```diff
@@ -97,6 +111,7 @@ async def generate(
         temperature=0.8,
         repetition_penalty=1.0,
         ignore_eos=False,
+        do_preprocess=True,
```

Comment:
It's not necessary. 
According to https://platform.openai.com/docs/api-reference/embeddings/create,
This API is used to creates an embedding vector representing **the input text.**
So I think we shouldn't decorate `prompt` at all.

Reply:
This is for inner usage. Both `/v1/completions` and `/v1/chat/completions` can use this function.